CustomPlugin/ServiceClientFactory.cs

```diff
@@ -27,14 +27,15 @@ public IServiceProxy GetDirectServiceProxy(string serviceName, InitServiceOption
 
         public IServiceProxy GetServiceProxy(InitServiceOptions options)
         {
-            var microserviceNameKey = _miaEnvConfiguration.MICROSERVICE_GATEWAY_SERVICE_NAME;
-            return string.IsNullOrEmpty(microserviceNameKey)
+            var microserviceName = _miaEnvConfiguration.MICROSERVICE_GATEWAY_SERVICE_NAME;
```

Comment:
```suggestion
            var microserviceGatewayName = _miaEnvConfiguration.MICROSERVICE_GATEWAY_SERVICE_NAME;
```

same below, microserviceName could be misleading